app/controllers/participant_groups_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ParticipantGroupsController < ApplicationController
+
+  load_and_authorize_resource
+
+  # PATCH/PUT /participant_group/1/group
+  def update
+    if @participant_group.update_attributes(group_param)
+      redirect_to :back, notice: I18n.t('participant_groups.update.successful') rescue ActionController::RedirectBackError redirect_to root_path
+    else
+      redirect_to :back, alert: I18n.t('participant_groups.update.failed') rescue ActionController::RedirectBackError redirect_to root_path
+    end
+  end
+
+  private
+
+    def group_param
```

Comment:
Well, `participant_group_params` doesn't exist anymore, and `group_param` is for all the parameters, there just happens to be only one :grin: 
So I would rename it to `participant_group_params`, but that's just as an advice, I will still accept the PR :wink: